docs/gen_docs.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
```

Comment:
Would you consider saying `set -e` here?

Reply:
will do.